waspc/packages/deploy/src/providers/fly/index.ts

```diff
@@ -20,6 +20,10 @@ class FlyCommand extends Command {
 			.option('--initial-cluster-size <initialClusterSize>', 'flyctl postgres create option', '1')
 			.option('--volume-size <volumeSize>', 'flyctl postgres create option', '1');
 	}
+	addLocalBuildOption(): this {
+		return this.option('--build-server-locally', 'force the server Docker container to build locally', false)
```

Comment:
What happens otherwise? It always get's built remotely?
When I read "forces", it sounds like normally it might build locally or it might build remotely, but now we made sure it builds locally -> but is that correct?
We could mention default behaviour here if there is one.

Reply:
The current implementation defaults to building remotely, which is mentioned in the docs (I can mention here too). This flag forces local. The default behavior of `flyctl deploy` without any flags is to build locally _if_ Docker was running, but otherwise build remotely. I am not a big fan of that for Wasp users to help support and debug build failures, etc., so was thinking it may benefit us to make it 100% deterministic, so we just default to remote but they can override and build locally if the need arises. Does that make sense?

Reply:
Actually, to probably better address your point, I can make it read something like `builds the Docker container locally instead of remotely` or something like that?